js/src/views/Application/Extension/store.js

```diff
@@ -68,6 +68,19 @@ export default class Store {
   installExtension = () => {
     this.setInstalling(true);
 
+    if (window.location.hostname === 'home.web3.site') {
```

Comment:
Should probably just check for <anything).web3.site, however it is good getting around the issue until we figure out a way forward.